docs/resources/private_endpoint_connection.md

```diff
@@ -6,6 +6,14 @@ description: |-
   Private endpoint connections allow customer applications to connect to a CockroachDB Cloud cluster without traversing the public internet. All application-database traffic remains within the cloud-provider network.
 ---
 
+# _Warning_: Use of *private endpoints* requires >=v1.7.6
+If you intend to use this provider to provision [private endpoints](https://github.com/cockroachdb/terraform-provider-cockroach/blob/main/docs/resources/private_endpoint_connection.md):
```

Comment:
The docs are generated from the output in the resources.  You'll need to update [here](https://github.com/cockroachdb/terraform-provider-cockroach/blob/main/internal/provider/private_endpoint_connection_resource.go#L51) and [here]( https://github.com/cockroachdb/terraform-provider-cockroach/blob/main/internal/provider/private_endpoint_services_resource.go#L45). and then run `make generate` to build the docs.

Reply:
ok thanks!